MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the Switch Editor thumb color
+
+#### Using event
+
+By default switch editor Thumb color is Blue, you can change the Thumb color by using `ThumbColor` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+            switchItem. ThumbColor = Colors.Pink;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the Switch editor background color
+
+#### Using event
+
+By default switch editor background colour is white, you can change the Background color by using `OnColor` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+            dataFormSwitchItem. OnColor = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+#### Changing the ItemsSource of combo box
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the `IDataFormSourceProvider`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight C# %}
+
+    public string Country { get; set; }
+    
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            
+            if (sourceName == "Country")
+            {
+                List<string> list = new List<string>()
+                {
+                    "India",
+                    "USA",
+                    "Japan"
+                };
+                return list;
+            }
+            return new List<string>();
+        }
+    }
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("Country", DataFormEditorType.ComboBox);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using GenerateDataFormItem event
+
+You can also set the ItemsSource for combo box editor by using the `GenerateDataFormItem` event in the `DataFormComboBoxItem`.
```

Comment:
You can also set the ItemsSource for combo box editor by using the `GenerateDataFormItem` event.

Reply:
updated